src/commands/opsGenieCommands.ts

```diff
@@ -1,50 +1,39 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import got from 'got'
 
-export async function createOpsGenieAlert(message: string, alias: string, apiKey: string, accountId: string) {
-  const options = {
-    json: {
-      message: message,
-      priority: 'P5',
-      alias: alias,
-      details: { AWSAccountId: accountId }
-    },
-    headers: header(apiKey)
-  }
+const baseURL = 'https://api.eu.opsgenie.com/v2/alerts?sort=updatedAt&order=asc'
+const pageSize = 20
 
-  const URL = 'https://api.eu.opsgenie.com/v2/alerts/'
-  const data: any = await got.post(URL, options).json()
+export async function getOpsGenieAlertsFromPoint(apiKey: string, lastUpdate: string, offset: number) {
+  const URL = `${baseURL}&limit=${pageSize}&offset=${offset * pageSize}&query=updatedAt>${dateParse(lastUpdate)}`
+  const response: any = await got(URL, options(apiKey))
 
-  return data
+  const insertTransactions = JSON.parse(response.body).data
+  const hasMore = 'next' in JSON.parse(response.body).paging
```

Comment:
```suggestion
  const responseBody = JSON.parse(response.body)
  const insertTransactions = responseBody.data
  const hasMore = 'next' in responseBody.paging
```

JSON.parse is slow - only do it once and then access the objects.
It also makes it less readable